src/test/java/org/sonar/etendo/java/EtendoJavaRulesDefinitionTest.java

```diff
@@ -0,0 +1,68 @@
+package org.sonar.etendo.java;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+import org.sonar.api.rules.RuleType;
+import org.sonar.api.server.rule.RuleParamType;
+import org.sonar.api.server.rule.RulesDefinition;
+import org.sonar.api.server.rule.RulesDefinition.Param;
+import org.sonar.api.server.rule.RulesDefinition.Repository;
+import org.sonar.api.server.rule.RulesDefinition.Rule;
+import org.sonar.etendo.java.checks.UseStringBuilderInsteadOfConcat;
+
+public class EtendoJavaRulesDefinitionTest {
+
+  private static final String CONCAT_RULE = "UseStringBuilderInsteadOfConcat";
+
+  private static void assertParameterProperties(Repository repository) {
+    Param threshold = repository.rule(CONCAT_RULE).param("Threshold");
+    assertThat(threshold).isNotNull();
+    String defaultThreshold = String.valueOf(UseStringBuilderInsteadOfConcat.DEFAULT_VALUE);
+    assertThat(threshold.defaultValue()).isEqualTo(defaultThreshold);
+    assertThat(threshold.description()).isEqualTo(
+        "Number of concatenations (literals and variables) that have to be made to trigger an issue");
```

Comment:
Maybe it's better to have all these kinds of messages created as constants.

Reply:
@RomanMagnoli done!